src/reflect/scala/reflect/api/Trees.scala

```diff
@@ -2463,7 +2464,9 @@ trait Trees { self: Universe =>
    *  @group Traversal
    */
   class Traverser {
```

Comment:
I wonder if all of these/some of these Traversers are thread local themselves.
Probably a bigger change thuogh  and higher risk